### ISSUE TITLE
Issues when downloading right after activating keys?

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/steam-for-linux#2575]
I bought the latest [Humble Bundle](http://www.humblebundle.com/), and got a Steam key for _Hotline Miami_ (this could be a title issue, so it's important).

I noticed that I got _no estimate_ in the popup window of how big the game was.

Then I clicked install and the download and install were _instantaneous_.

I tried launching it, turned out the executable was missing. And indeed, I looked at the folder under `~/.steam/steam/SteamApps/common/hotline_miami` and it was _completely empty_.

After removing and re-installing several times, I found out that I should 'verify local game data integrity' under 'Properties'. It said it was okay.

**TL;DR:** game 'installs' but didn't download anything whatsoever. Obviously, doesn't work. Integrity falsely said to be okay.
